epub33/core/index.html

```diff
@@ -5529,6 +5529,11 @@ <h4>Path and File Names</h4>
 						</li>
 					</ul>
 
+					<div class="note">
+						<p>EPUB Creators may have to truncate File Names to keep it within the 255 bytes range. When doing so it is important to avoid mid-character truncation due to the difference between bytes and characters in multibyte encodings such as UTF-8. See section on <a data-cite="international-specs/#char_truncation">Truncating or limiting the length of strings</a> [[international-specs]] for more information. </p>
```

Comment:
probably better: truncate File Names to keep it -> truncate a File Name to keep it
(not sure about the capitalisation - assuming this is intentional)

Reply:
It is so capitalized in the full section.

You were too quick for me:-) Right after the submission of the PR I realized this and the text now says:

> File Names to keep them within the 255 bytes range